examples/06_bmg_flash_attention/06_xe_fmha_fwd.cpp

```diff
@@ -107,36 +107,45 @@ int main(int argc, const char **argv) {
 
 #endif
 #elif defined(DECODE)
+
+#if PERSISTENT
+#define NUM_SG _16
+#define KV_TILE_SIZE _256
+#else
+#define NUM_SG _16
```

Comment:
Did you mean to change the tile configuration for non-persistent mode?
```suggestion
#define NUM_SG _8
```

Reply:
Changed back to `_8` now.